### ISSUE TITLE
Quiz Seats and Officials' Table Setup

### ISSUE THREAD

**Issue by scottpeterson** [gryphonshafer/Quizzing-Rule-Book#70]
Because a straight line of 12 seats creates a different viewing angle, especially for those at the end, there should be a best practice "arc degrees" when using individual seats, and a best practice "angle between benches" when using benches, to make the viewing angles between quizzer and QM as equal and fair as possible for all 12. 

Similarly, the distance between quiz jump equipment and the Officials' Table, specifically the QM, should have a best practice distance. 

This will also minimize the time it takes a quizzer to move from their seat to the mic (which is the optimal answering location, though not required at all). 

Also, in the quizzing world we first return to, there may be COVID precautions that need to be taken (like answering within a certain spot). This spot should also be chosen to be fair to all 12 participants. 

All of these should be best practices, since Quiz Room logistics may make the best practice impossible.

**Comment by jttower** [comment]
Should there be a related "best practices" statement about the seating of coaches in relation to the QM table?  (In line with, how far from the side of the QM table, behind a certain number of feet, etc.) Again, space limitations may make this a moot point, so I'm not even sure if it is necessary to delineate these types of things.

**Comment by scottpeterson** [comment]
Potentially. 

Initially, as a QM, I didn't want coaches able to see my computer screen and timer. 

Increasingly, I think that should be encouraged. 

As Gryphon mentioned on #72, making the timer public to all would be a positive, and would encourage and enforce QMs to be accurate and consistent.

**Comment by jswingle** [comment]
I'm mostly indifferent as to whether this is really necessary, but I'm not _against_ it for a best practices document. Most of the time there are space limitations and constraints, even at Internationals. Benches are standard sizes and there's only so much you can do with metal folding chairs.

**Comment by ARMediting** [comment]
A rule about distance between coaches and the officials and between coaches and quizzers seems prudent (as it provides a better experience for the quizzers), but I feel like attempting a good arc arrangement is obvious.

**Comment by gryphonshafer** [comment]
I like the intent here, but I'm worried if it gets into a strict rule, we'll have to intentionally violate it given some situations. I'm totally cool with it being a best practice, though.

**Comment by jswingle** [comment]
@scottpeterson @jttower @ARMediting All of you have expressed varying degrees of interest in this as a rule or best practice, anyone want to write some language concerning it? I'm still in the "meh, don't exactly see the point" phase as I posted above, but I would vote in favor of a good proposal.

**Comment by scottpeterson** [comment]
Best Practices:

The ideal distance between the officials table and the mid-point of quiz seats should be 14-22 feet. 

The ideal angle between each bench should be 120 degrees. 


I just kinda "mind-balled" those numbers.

**Comment by josiah-leinbach** [comment]
In the real world, there are far too many variables with a room arrangement to make any such rule viable. And besides that, I have never a quiz room in which people set up benches or chairs in a straight line anyway. People recognize intuitively that angling toward the officials table is good anyway.

Plus, for what its worth, I think if we make tape-measures and angle-finders required quiz equipment, we have will have made quizzing less accessible and less attractive for new programs, which will hamstring any efforts to grow quizzing.

**Comment by jswingle** [comment]
Yeah I personally don't think this is a necessary change even for a best practices, though I do appreciate Scott's "mind-balling" above. From what I've experienced, every meet I've been at has done the most possible to make an ideal quizzing setup given the limitations of the venue. Nobody directs and sets up a meet without some sort of sense of how it ought to be done.

I'm for closing the issue out.
